.github/workflows/ci.yaml

```
@@ -848,6 +848,10 @@ jobs:
       - name: Register pytest slow test problem matcher
         run: |
           echo "::add-matcher::.github/workflows/matchers/pytest-slow.json"
+      - name: Compile translations
```

Maybe clarify this build the English translations?
```suggestion
      - name: Compile English translations
```